src/main/resources/db/migration/V1_78__move_appointments_to_new_action_plan_sessions.sql

```diff
@@ -0,0 +1,65 @@
+create table action_plan_session_appointment_pre_v1_78 as select * from action_plan_session_appointment;
+
+with action_plans_to_modify as (
+    with referrals_with_multiple_approved_aps as (
+        select
+            r.id referral_id
+        from
+            referral r
+        where
+            r.id in (
+                     '05b1d52a-ae5c-49a6-8b4d-080a5101e79f',
+                     '0be1d4f5-53a1-4e11-8c35-803774f67c7a',
+                     '68e20461-6a99-4046-a63f-657c9f434da0',
+                     '75d3a978-fefc-4252-84d1-45fa86d27b90',
+                     '9d63bc91-ac1f-4a62-900a-08e826fedf55',
+                     'd2d72900-a5db-4711-a0ab-d48718f5af1d',
+                     'd60d1343-2e3f-44cb-93d4-3280938e7824',
+                     'da8edf26-d831-4b3d-bd00-9618ca4a9fa3'
+                    )
+    ),
+         appointments_per_action_plan as (
+             select
+                 ap.referral_id,
+                 ap.id action_plan_id,
+                 ap.approved_at,
+                 count(aps.id) session_count,
```

Comment:
Just checking: is `ap.number_of_sessions` different?

Reply:
It should be the same. 
To be honest - not sure why we have that field since we can obtain it from counting the sessions on an action plan.

Reply:
Well, we only generate the sessions when the plan is approved

Reply:
Ok - but it shouldn't matter if we use one or the other, right?